src/common/contentPreference.ts

```diff
@@ -247,7 +246,7 @@ const unfollowSource: UnFollowEntity = async ({ ctx, id }) => {
 
     await repository.delete({
       userId: ctx.userId,
-      sourceId: id,
```

Comment:
Dont you need these still?

Reply:
no, it was a mistake from before, referenceId is sourceId in this case and also PK so delete is faster

Reply:
Ah but whats the denominator between source-tag then?

Matbe im not seeing it on the changeset though

Reply:
Ah that, typeorm automatically adds `type` where clause because I am using `ContentPreferenceSource` entity

Reply:
Ah i see, that answers it 🙏